src/API/OnboardingTasks.php

```diff
@@ -810,19 +794,10 @@ public function dismiss_task( $request ) {
 	 * @return WP_REST_Request|WP_Error
 	 */
 	public function undo_dismiss_task( $request ) {
-		$id = $request->get_param( 'id' );
-
-		$dismissed = get_option( 'woocommerce_task_list_dismissed_tasks', array() );
-		$dismissed = array_diff( $dismissed, array( $id ) );
-		$update    = update_option( 'woocommerce_task_list_dismissed_tasks', $dismissed );
-
-		if ( $update ) {
-			wc_admin_record_tracks_event( 'tasklist_undo_dismiss_task', array( 'task_name' => $id ) );
-		}
-
+		$id   = $request->get_param( 'id' );
 		$task = TaskLists::get_task( $id );
```

Comment:
Also here, for handling the case of an invalid task ID.

Reply:
Also handled, but same comment as above.